### DISCUSSION TITLE
error SMTP

### DISCUSSION THREAD

**Discussion by Geeckonnect** [authelia/authelia#3408] [Q&A] [answered]
Hello, i have a problem to configuring authelia with an SMTP notifier, it works with a notification.txt, 
Here's my log error 

time="2022-05-19T17:11:02+03:00" level=debug msg="Notifier SMTP client attempting connection to 192.168.98.3:25"
time="2022-05-19T17:11:02+03:00" level=debug msg="Notifier SMTP client connected successfully"
time="2022-05-19T17:11:02+03:00" level=debug msg="Notifier SMTP server supports STARTTLS (disableVerifyCert: false, ServerName: 192.168.98.3), attempting"
time="2022-05-19T17:11:02+03:00" level=warning msg="Notifier SMTP client encountered error during cleanup: x509: certificate has expired or is not yet valid: current time 2022-05-19T17:11:02+03:00 is after 2018-06-11T06:59:49Z"
time="2022-05-19T17:11:02+03:00" level=error msg="Failure running the notification provider startup check: x509: certificate has expired or is not yet valid: current time 2022-05-19T17:11:02+03:00 is after 2018-06-11T06:59:49Z" stack="github.com/authelia/authelia/v4/internal/commands/root.go:106 doStartupChecks\ngithub.com/authelia/authelia/v4/internal/commands/root.go:78  cmdRootRun\ngithub.com/spf13/cobra@v1.4.0/command.go:860                  (*Command).execute\ngithub.com/spf13/cobra@v1.4.0/command.go:974                  (*Command).ExecuteC\ngithub.com/spf13/cobra@v1.4.0/command.go:902                  (*Command).Execute\ngithub.com/authelia/authelia/v4/cmd/authelia/main.go:10       main\nruntime/proc.go:250                                           main\nruntime/asm_amd64.s:1571                                      goexit"
time="2022-05-19T17:11:07+03:00" level=warning msg="Could not read from the NTP server socket to validate the system time is properly synchronized: read udp 172.18.0.4:44626->162.159.200.123:123: i/o timeout"
time="2022-05-19T17:11:07+03:00" level=fatal msg="The following providers had fatal failures during startup: notification" stack="github.com/authelia/authelia/v4/internal/commands/root.go:122 doStartupChecks\ngithub.com/authelia/authelia/v4/internal/commands/root.go:78  cmdRootRun\ngithub.com/spf13/cobra@v1.4.0/command.go:860                  (*Command).execute\ngithub.com/spf13/cobra@v1.4.0/command.go:974                  (*Command).ExecuteC\ngithub.com/spf13/cobra@v1.4.0/command.go:902                  (*Command).Execute\ngithub.com/authelia/authelia/v4/cmd/authelia/main.go:10       main\nruntime/proc.go:250                                           main\nruntime/asm_amd64.s:1571                                      goexit"

Thanks for your reply , i need some help

**Comment by james-d-elliott** [comment, marked as answer]
The SMTP server certificate is expired, expired `2022-05-19T17:11:02+03:00`
